terragrunt/aws/database/documentdb.tf

```diff
@@ -0,0 +1,41 @@
+#
+# Terraform code to create an Amazon DocumentDB cluster 
+#
+
+module "cra_upd_documentdb" {
+  source = "github.com/cds-snc/terraform-modules//DocumentDB?ref=v10.4.4"
+
+  database_name          = "upd"
+  billing_code           = var.billing_code
+  master_username        = var.secrets["DOCDB_USERNAME"]
```

Comment:
I would not put this in a variable that you build in during the environment.hcl setup since you might inadvertently leak them.  I would put any secrets you need to store in a secrets manage such as ssm parameters. An example for document db is here -https://github.com/cds-snc/ai-answers/blob/main/terragrunt/aws/database/documentdb.tf

Reply:
My logic here was since the secrets are already marked as sensitive and are passed to SSM in exactly the same way as I did here, using SSM for this would essentially just be an extra round trip that doesn't actually change anything. You can see in the `terragrunt plan` output that the values are redacted.

That said, it's not a big deal, I'll make the changes.